deploy/helm/galaxy/values.yaml

```diff
@@ -10,24 +10,46 @@ image:
 # -- This is a cron expression that defines the schedule for the cronjob
 schedule: "*/59 * * * *" # Every 59 minutes
 
-# -- Environment variables to be set in the container
+# -- External secrets configuration
+externalSecrets:
+  # -- Enable external secrets
+  enabled: false
+  # -- The name of the external secrets
+  target: "my-vault-secrets"
+  # -- All keys in secrets file will be exported as environment variables
+  allAsEnv: false
+  # -- environment variables to be set in the container from the external secrets if allAsEnv is false
+  # envs is an array of objects with the following keys
+  # name -- The name of the environment variable to setup
+  # key -- The key in the secret to use
+  # eg.:
+  # envs:
+  # - name: "RELY_API_TOKEN"
+  #   key: "api_token"
+  envs: []
+
+# -- The configuration for the integration
+integration:
+  # -- The type of the integration
+  # can be any of the following: pagerduty, github, gitlab, bitbucket, sonarqube, aws, opsgenie, gcp
```

Comment:
I think it's easy to forget to keep this list up to date

Could we point to a documentation link instead?

Reply:
Makes sense, I don't think we have a list in the docs yet, but should  be changed latter I gess

Reply:
We can put this in the readme file and later add it to docs.rely.io. We still haven't covered it there.